### ISSUE TITLE
Feature - read_json()

### ISSUE THREAD

**Issue by karishmashuklaa** [karishmashuklaa/ickle#23]
Read JSON files in ickle dataframes.
Can load either JSON files or Python dictionaries (as JSON and dictionaries have the same format)

**Comment by tharun634** [comment]
Hello, I am interested in this issue.

Can you give me a JSON file / Python dictionary example of an input to `read_json()`. Because, I see that the `__init__` takes a dict as well, so I just wanted to understand how `read_json()` will be different from `__init__`

**Comment by karishmashuklaa** [comment]
@tharun634 Hello. You are right - for dictionaries we simply create a dataframe and print it. We don't need to handle it in `read_json()` method. Our primary focus is JSON file. 
Sample JSON file - http://jsonblob.com/1025743492709892096

**Comment by tharun634** [comment]
Thanks for the response.

So, looking from the JSON blob that you shared. Will there be an array of such JSON objects in the `json` file? such as [this](https://jsonblob.com/1025745788843868160)? Where each JSON Object in the array corresponds to a row in the dataframe

**Comment by karishmashuklaa** [comment]
@tharun634 Yes - possible. We can first go with a simple JSON files and then dig deeper.
Also checkout `json` package in built in Python. Can be of help 😄 https://docs.python.org/3/library/json.html
